### PR TITLE
chore: Drop `pinToSuperview(axis anchor`, `alignCenter`, `centerInSuperView`, `showsSelectionIndicator` IC - 73

### PR DESCRIPTION
**Pull request by agisilaos**
----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [x] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

With this PR we drop `pinToSuperview(axis anchor`, `alignCenter`, `centerInSuperView`, `showsSelectionIndicator` because of their deprecation warnings.

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Components/Loading/ProgressSpinner.swift [hunk 1]
```diff
@@ -114,7 +114,8 @@ final class ProgressSpinner: UIView {
 
     private func setupConstraints() {
         spinner.translatesAutoresizingMaskIntoConstraints = false
-        spinner.centerInSuperview()
+        spinner.centerXAnchor.constraint(equalTo: centerXAnchor, constant: 0).isActive = true
+        spinner.centerYAnchor.constraint(equalTo: centerYAnchor, constant: 0).isActive = true
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Components/Loading/ProgressSpinner.swift:118]: If the constant is 0 I think you can omit it.

File: Wire-iOS/Sources/UserInterface/Components/Loading/Spinner.swift [hunk 2]
```diff
@@ -85,7 +85,8 @@ final class Spinner: UIView {
 
     private func setupConstraints() {
         spinner.translatesAutoresizingMaskIntoConstraints = false
-        spinner.centerInSuperview()
+        spinner.centerXAnchor.constraint(equalTo: centerXAnchor, constant: 0).isActive = true
+        spinner.centerYAnchor.constraint(equalTo: centerYAnchor, constant: 0).isActive = true
```
Comment (johnxnguyen) [inline Wire-iOS/Sources/UserInterface/Components/Loading/Spinner.swift:89]: If the constant is 0 I think you can omit it.

**Review by johnxnguyen — APPROVED** [review]

File: Wire-iOS/Sources/UserInterface/Components/Loading/Spinner.swift (hunk 2 above)
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Components/Loading/Spinner.swift]: Done!

File: Wire-iOS/Sources/UserInterface/Components/Loading/ProgressSpinner.swift (hunk 1 above)
Comment (agisilaos) [reply Wire-iOS/Sources/UserInterface/Components/Loading/ProgressSpinner.swift]: Done!

File: Wire-iOS/Sources/UserInterface/Sketchpad/SketchColorCollectionViewCell.swift [hunk 3]
```diff
@@ -72,7 +72,8 @@ final class SketchColorCollectionViewCell: UICollectionViewCell {
         }
 
         knobView.translatesAutoresizingMaskIntoConstraints = false
-        knobView.centerInSuperview()
+        knobView.centerXAnchor.constraint(equalTo: centerXAnchor, constant: 0).isActive = true
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Sketchpad/SketchColorCollectionViewCell.swift:75]:
```suggestion
        knobView.centerXAnchor.constraint(equalTo: centerXAnchor).isActive = true
```

File: Wire-iOS/Sources/UserInterface/Sketchpad/SketchColorCollectionViewCell.swift [hunk 4]
```diff
@@ -72,7 +72,8 @@ final class SketchColorCollectionViewCell: UICollectionViewCell {
...
+        knobView.centerYAnchor.constraint(equalTo: centerYAnchor, constant: 0).isActive = true
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Sketchpad/SketchColorCollectionViewCell.swift:76]:
```suggestion
        knobView.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
```

File: Wire-iOS/Sources/UserInterface/StartUI/Common/EmptySearchResultsView.swift [hunk 5]
```diff
@@ -110,7 +110,8 @@ final class EmptySearchResultsView: UIView {
 
         addSubview(stackView)
 
-        stackView.centerInSuperview()
+        stackView.centerXAnchor.constraint(equalTo: centerXAnchor, constant: 0).isActive = true
+        stackView.centerYAnchor.constraint(equalTo: centerYAnchor, constant: 0).isActive = true
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/StartUI/Common/EmptySearchResultsView.swift:114]:
```suggestion
        stackView.centerYAnchor.constraint(equalTo: centerYAnchor).isActive = true
```

File: Wire-iOS/Sources/UserInterface/Settings/ProfileSelfPictureViewController.swift [hunk 6]
```diff
@@ -99,7 +99,8 @@ final class ProfileSelfPictureViewController: UIViewController {
             bottomOffset = -UIScreen.safeArea.bottom + 20.0
         }
 
-        cameraButton.alignCenter(to: bottomOverlayView, with: CGPoint(x: 0, y: bottomOffset))
+        cameraButton.centerXAnchor.constraint(equalTo: bottomOverlayView.centerXAnchor, constant: 0).isActive = true
```
Comment (KaterinaWire) [inline Wire-iOS/Sources/UserInterface/Settings/ProfileSelfPictureViewController.swift:102]:
```suggestion
        cameraButton.centerXAnchor.constraint(equalTo: bottomOverlayView.centerXAnchor).isActive = true
```

**Pull request merged**